pkg/chunkserver/chunkfilepool.go

```diff
@@ -17,8 +17,8 @@ import (
 )
 
 const (
-	PrepareJobName         = "prepare-chunkfile"
-	DEFAULT_CHUNKFILE_SIZE = 16 * 1024 * 1024 // 16MB
```

Comment:
For numbers, uppercase should be better, but you can modify the initials of 'PrepareJobName' and other variables in another package , because other packages do not use this value.

Reply:
It's not Go's convention, but it's okay to make it a convention for this repository.

Reply:
`PrepareJobName` is used in controllers, so I can't change it to be unexported.

Reply:
Reverted. PTAL